### ISSUE TITLE
auro-button: Focus outlines missing on Windows High Contrast mode

### ISSUE THREAD

**Issue by geoffrich** [AlaskaAirlines/auro-button#89]
## Describe the bug
When Windows High Contrast Mode is enabled, focus outlines are not shown when tabbing.

![image](https://user-images.githubusercontent.com/4992896/98700163-be1c7000-232c-11eb-9519-b3f86c3639a6.png)

I believe this is because WHCM [ignores border or background color changes and box shadows for focus states](https://sarahmhigley.com/writing/whcm-quick-tips/#1.-custom-focus-styles-%2B-a-transparent-outline). 

## Desktop (please complete the following information):
 - OS: Windows 10
 - Browser: Firefox

## Additional context
I think we should update the default focus styles in WCSS and inherit the changes here.

**Comment by geoffrich** [comment]
FWIW, this mainly seems to be a Firefox issue. Edge seems to apply a focus outline automatically:

![image](https://user-images.githubusercontent.com/4992896/98718685-971d6880-2343-11eb-825c-afa23abce9ba.png)

**Comment by blackfalcon** [comment]
Think it would be best if the same solution used in https://github.com/AlaskaAirlines/auro-radio/pull/8 was used here.
